<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="WorkflowApplicationCompletedException.xml" source-language="en-US" target-language="zh-TW">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac59640587b33d8a415f12e8ff91bf8ef99a7124dbe.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">9640587b33d8a415f12e8ff91bf8ef99a7124dbe</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Activities.WorkflowApplicationCompletedException">
          <source>Represents the exception that is thrown when an operation on a workflow instance is not valid because the instance has completed.</source>
          <target state="translated">表示當工作流程執行個體上的作業因為執行個體完成而無效時所擲回的例外狀況。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Activities.WorkflowApplicationCompletedException">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class.</source>
          <target state="translated">建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class.</source>
          <target state="translated">建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String)">
          <source>A description of the application completed exception.</source>
          <target state="translated">應用程式完成的例外狀況的描述。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String)">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class with a text string description of the exception.</source>
          <target state="translated">使用例外狀況的文字字串描述，建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>All the information required to serialize or deserialize the application completed exception object.</source>
          <target state="translated">序列化或還原序列化應用程式完成的例外狀況物件所需的所有資訊。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The source and destination of a given serialized stream and an additional caller-defined context.</source>
          <target state="translated">指定之序列化資料流的來源和目的端，和其他呼叫端定義的內容。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class with the context and serialization information required to retrieve data about this exception from a serialized stream.</source>
          <target state="translated">使用擷取自序列化資料流且與此例外狀況有關的資料時所需的內容和序列化資訊，建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)">
          <source>A description of the application completed exception.</source>
          <target state="translated">應用程式完成的例外狀況的描述。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)">
          <source>An exception that has a causal relationship to the application completed exception.</source>
          <target state="translated">與應用程式完成的例外狀況有因果關係的例外狀況。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)">
          <source>The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.</source>
          <target state="translated">已擲回應用程式完成的例外狀況以回應內部例外狀況，且可以使用內部例外狀況的資訊以更適當地處理錯誤。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class with a text string description of the exception and with an additional exception that is the cause of the application completed exception.</source>
          <target state="translated">使用例外狀況的文字字串描述和造成此應用程式完成的例外狀況之其他例外狀況，建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)">
          <source>A description of the application completed exception.</source>
          <target state="translated">應用程式完成的例外狀況的描述。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)">
          <source>A globally unique identifier.</source>
          <target state="translated">全域唯一識別碼。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class with a text string description of the exception and with a unique identifier for the exception.</source>
          <target state="translated">使用例外狀況的文字字串描述和唯一識別碼，為這個例外狀況建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)">
          <source>A description of the application completed exception.</source>
          <target state="translated">應用程式完成的例外狀況的描述。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)">
          <source>A globally unique identifier.</source>
          <target state="translated">全域唯一識別碼。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)">
          <source>An exception that has a causal relationship to the application completed exception.</source>
          <target state="translated">與應用程式完成的例外狀況有因果關係的例外狀況。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)">
          <source>The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.</source>
          <target state="translated">已擲回應用程式完成的例外狀況以回應內部例外狀況，且可以使用內部例外狀況的資訊以更適當地處理錯誤。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> class with a text string description of the exception, a unique identifier for it, and an additional exception that is the cause of the application completed exception.</source>
          <target state="translated">使用例外狀況的文字字串描述、其唯一識別碼和造成此應用程式完成之例外狀況的其他例外狀況，建立 <ph id="ph1">&lt;see cref="T:System.Activities.WorkflowApplicationCompletedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>